lib/main.dart

```diff
@@ -31,7 +31,7 @@ class MyApp extends StatelessWidget {
         colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
         useMaterial3: true,
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: const MyHomePage(title: 'Hello World'),
```

Comment:
should replace 'Hello World' by 'Hello CQ2021'

Reply:
I updated with your review